packages/angular-cli/blueprints/ng2/files/__path__/app/app.component.spec.ts

```diff
@@ -1,11 +1,15 @@
 /* tslint:disable:no-unused-variable */
 
-import { TestBed, async } from '@angular/core/testing';
+import { TestBed, async } from '@angular/core/testing';<% if (routing) { %>
+import { RouterTestingModule } from '@angular/router/testing';<% } %>
 import { AppComponent } from './app.component';
 
 describe('AppComponent', () => {
   beforeEach(() => {
-    TestBed.configureTestingModule({
+    TestBed.configureTestingModule({<% if (routing) { %>
+      imports: [
+        RouterTestingModule
+      ],<% } %>
```

Comment:
While we're fixing this, can you add a `compileComponents` call in here as well?